### PR TITLE
Bugfix/209 response error interceptor loop

### LINKED ISSUE #209: Bad token causes infinite validation loop
The `responseError` http interceptor triggers the auth flow when a 401 response is received. The first step of the response flow is to validate a token if one exists. If a token exists but is invalid, we get a 401 response, which triggers the interceptor. This repeats forever.

image

As in the solution center interceptor, we should distinguish calls to the token service and not trigger the auth flow if the 401 call came from that service.

### PR DESCRIPTION
**Pull request by cjhowald**
Fixes #209 

Test in user management.

### CODE REVIEW

File: src/app/common/security/interceptor.service.spec.js [hunk 1]
```diff
@@ -36,20 +37,28 @@ describe('Interceptor service', () => {
   });
 
   describe('responseError', () => {
-    it('should do nothing for status not 401', () => {
-      let resolved = true;
-      interceptorService.responseError(mock.responseError500).catch(response => {
-        resolved = false;
-        expect(response).toBe(mock.responseError500);
-      });
+    it('should do nothing for status not 401', (done) => {
```
Comment (dami-gg) [inline src/app/common/security/interceptor.service.spec.js]: Nice way to handle it

**Comment by dami-gg** [comment]
Looks good and all the tests pass

**Comment by dami-gg** [comment]
:+1:

**Comment by cjhowald** [comment]
👍

**Pull request merged**